PHP/CodeSniffer/Standards/Sentact/ruleset.xml

```diff
@@ -2,10 +2,19 @@
 <ruleset name="Sentact">
     <description>The Sentact coding standard.</description>
 
-    <rule ref="PSR2"/>
+    <rule ref="PSR2">
+        <exclude name="PSR1.Classes.ClassDeclaration.MissingNamespace"/>
+        <exclude name="Squiz.Classes.ValidClassName.NotCamelCaps"/>
+    </rule>
 
     <!-- Proper docblock formatting -->
     <rule ref="Generic.Commenting.DocComment"/>
+    <rule ref="Generic.Commenting.DocComment.ShortNotCapital">
```

Comment:
What are these for?

Reply:
When you add descriptive info to a doc comment like outlining data formats, etc. the code sniffer will complain that something like 

`array(123)` will not start with an upper case

Reply:
Ah, yeah. It's trying to encourage full sentences. I'm fine with taking that out.